querybook/server/logic/schedule.py

```diff
@@ -180,7 +182,7 @@ def get_scheduled_data_docs_by_user(
         session.query(DataDoc, TaskSchedule)
         .join(
             TaskSchedule,
-            TaskSchedule.name == func.concat("run_data_doc_", DataDoc.id),
+            TaskSchedule.name == func.concat(DATADOC_SCHEDULE_PREFIX, DataDoc.id),
```

Comment:
why not just use `get_data_doc_schedule_name` ^

Reply:
DataDoc.id is not a string, its a placeholder variable in SQL